.travis.yml

```diff
@@ -9,9 +9,9 @@ env:
     #               dependencies are supposed to be skipped
     # Note that we don't run coverage on Py3k because it slows our tests
     # by a factor of 2 (!)
-    - PYTHON=2.7 DEPS=full TEST_LOCATION=src MNE_DONTWRITE_HOME=true
+    - PYTHON=2.7 DEPS=full TEST_LOCATION=src
     - PYTHON=2.7 DEPS=full TEST_LOCATION=install
-    - PYTHON=2.7 DEPS=nodata TEST_LOCATION=src  # also runs flake8
+    - PYTHON=2.7 DEPS=nodata TEST_LOCATION=src MNE_DONTWRITE_HOME=true  # also runs flake8
```

Comment:
Also FYI this fixes our coverage. Turns out the 2.7 full deps src version wasn't actually running any of the tests that required testing data, so after this PR our coverage should jump back up to 90+.